graphql_compiler/helpers.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2019-present Kensho Technologies, LLC.
+import funcy
```

Comment:
Try naming this file something different than `helpers` so we don't have ambiguous imports of `helpers` depending on how many levels up in the hierarchy one goes. How about `global_utils`?